Support/Text/BytePairEncoder.swift

```diff
@@ -62,21 +67,25 @@ public struct BytePairEncoder {
     public func encode(token: String, variant: Variant? = .roberta) -> [String] {
         // if let cached = cache[token] { return cached }
         // let token = " " + token
-        var parts: [String]
-        var encodedParts: [String]
-        var encoded: [String]
+        var parts = [String]()
 
         switch variant {
         case .gpt2:
             // Split into parts before encoding.
-            parts = BytePairEncoder.splittingWithDelimiters(
+            let unencodedTokens: [String] = BytePairEncoder.splittingWithDelimiters(
```

Comment:
```suggestion
            let unencodedTokens = BytePairEncoder.splittingWithDelimiters(
```
Minor, but the type should be able to be inferred here.